toolkits/global/packages/global-pagination/README.md

```diff
@@ -0,0 +1,52 @@
+# Global Pagination
+
+A component for the controls of pagination.
+
+## Branding
+
+To include `global-pagination` component currently uses the `DEFAULT` brand only.
```

Comment:
Can you update this section to give an example - along the lines of this
https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-banner#branding

Reply:
Done!